apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.ConformanceTests/Services/CloudBigtableV2TestProxyImpl.cs

```diff
@@ -0,0 +1,393 @@
+        return new CreateClientResponse();
+
+        void MaybeOverrideTimeoutSetting(BigtableServiceApiSettings settings)
+        {
+            if (request.PerOperationTimeout is not null)
+            {
+                CallSettings callSettings = CallSettings.FromExpiration(Expiration.FromTimeout(request.PerOperationTimeout.ToTimeSpan()));
+                settings.CheckAndMutateRowSettings = callSettings;
+                settings.ExecuteQuerySettings = callSettings;
+                settings.MutateRowSettings = callSettings;
+                settings.MutateRowsSettings = callSettings;
+                settings.ReadRowsSettings = callSettings;
+                settings.ReadModifyWriteRowSettings = callSettings;
+                settings.SampleRowKeysSettings = callSettings;
+                settings.CallSettings = callSettings;
+            }
+        }
+
+        ChannelCredentials GetChannelCredentials()
+        {
+            // securityOptions.UseSsl, securityOptions.SslRootCertsPem, securityOptions.AccessToken
+            if (request.SecurityOptions.AccessToken is null)
+            {
+                return request.SecurityOptions.UseSsl ? new SslCredentials(request.SecurityOptions.SslRootCertsPem) : ChannelCredentials.Insecure;
+            }
+            GoogleCredential googleCredential = GoogleCredential.FromAccessToken(request.SecurityOptions.AccessToken);
+            return request.SecurityOptions.UseSsl
+                ? ChannelCredentials.Create(new SslCredentials(request.SecurityOptions.SslRootCertsPem), googleCredential.ToCallCredentials())
+                : googleCredential.ToChannelCredentials();
+        }
+
+        GrpcChannelOptions GetChannelOptions() => request.SecurityOptions.SslEndpointOverride is not null
+                    ? GrpcChannelOptions.Empty.WithCustomOption("grpc.ssl_target_name_override", request.SecurityOptions.SslEndpointOverride)
+                    : null;
+    }
+
+    public override async Task<CloseClientResponse> CloseClient(CloseClientRequest request, ServerCallContext context)
+    {
+        CbtClient cbtClient = GetClient(request.ClientId, context);
+        try
+        {
+            if (cbtClient.LastCreatedChannel is not null)
+            {
+                await cbtClient.LastCreatedChannel.ShutdownAsync();
+            }
+        }
+        catch (Exception e)
+        {
+            context.Status = new Status(StatusCode.Internal, e.Message, e);
+            throw new RpcException(context.Status);
+        }
+        context.Status = new Status(StatusCode.OK, "CloseClient succeeded");
+        return new CloseClientResponse();
+    }
+
+    public override Task<RemoveClientResponse> RemoveClient(RemoveClientRequest request, ServerCallContext context)
+    {
+        string clientId = request.ClientId;
+        GaxPreconditions.CheckArgument(clientId is not ("" or null), "ClientId", "client id must be provided", context);
```

Comment:
Just a reminder, remove all of these validations. Let's assume the text engine is passing the right test data.